src/sentry/static/sentry/app/components/events/interfaces/threads/threadsSelectorOption.tsx

```diff
@@ -0,0 +1,110 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
+import styled from '@emotion/styled';
+
+import space from 'app/styles/space';
+import TextOverflow from 'app/components/textOverflow';
+import InlineSvg from 'app/components/inlineSvg';
+import {EntryTypeData} from 'app/types';
+
+type Props = {
+  id: string;
+  name?: string;
+  crashed?: boolean;
+  details: ThreadInfo;
+  crashedInfo?: EntryTypeData;
```

Comment:
this is silly, but it's a good practice to list required props first and then the optional ones